data/snippet/cookie_extract.toml

```diff
@@ -0,0 +1,22 @@
+id = "extract_cookies"
+name = "Extract Cookie Value"
+description = "Extracts the value of a Cookie, given the Cookie header and name."
+code = """
+async function get_cookie(cookies, name) {
```

Comment:
See comments for Signed request snippet. 

Also must be in `handleRequest` format.
Open to opinions on revising [template contribution guide](https://github.com/cloudflare/workers-docs/blob/master/content/templates/CONTRIBUTING.md) as I realize for example like this it doesn't always make sense. Just want to find a way to be consistent